unlock-protocol-com/blog/guild-hook.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Guild.xyz hook
+subTitle: Easily create all kinds of allow-list on your lock using Guild.xyz!
+description: By leveraging a new kind of smart contract hook, you can easily create all kinds of allow-list on your lock using Guild.xyz.
+author: Julien Genestoux
+publishDate: June 15, 2023
+image: /images/blog/guild-hook/guild.xyz.jpeg
```

Comment:
@julien51 please swap for this image for brand consistency https://unlock-protocol.com/guides/wp-content/uploads/2022/06/guild.xyz_.png